includes/features/SU-SWS/stanford_private_page/stanford_private_page.feature

```diff
@@ -30,9 +20,8 @@ Feature: Stanford Private Page
     Then I go to "private/bar"
     And I should see "Bar" in the "Content Head" region
 
-  @api @javascript @safe
+  @api @safe
   Scenario: As a site owner I can see Create Private Page
```

Comment:
We should not be using the site owner role for testing here. Change to administrator.